scripts/atc_thehive/thehive_classes.py

```diff
@@ -112,7 +113,8 @@ def validate(self):
                 and self.order >= 0 \
                 and isinstance(self.title, str) \
                 and isinstance(self.group, str) \
-                and isinstance(self.description, str):
+                and isinstance(self.description, str) \
+                and isinstance(self.owner, str):
```

Comment:
Can we make `owner` a not mondatory field? I believe that it would be more flexible for those who are not (and not going to) using it. 

The rest looks perfect! (:

Reply:
This should be in place now